client/lib/plans/plans-list.js

```diff
@@ -1333,6 +1333,39 @@ export const PLANS_LIST = {
 		getPathSlug: () => 'complete-monthly',
 		getProductId: () => 2015,
 	},
+
+	[ constants.PLAN_P2_PLUS ]: {
+		group: constants.GROUP_WPCOM,
+		type: constants.TYPE_P2,
+		getTitle: () => i18n.translate( 'P2+' ),
+		getDescription: () => i18n.translate( 'Some long description' ),
+		getShortDescription: () => i18n.translate( 'Some short description' ),
+		getPlanCompareFeatures: () => [
+			// pay attention to ordering, shared features should align on /plan page
+			constants.FEATURE_EMAIL_SUPPORT,
+			constants.FEATURE_BASIC_DESIGN,
+			constants.FEATURE_13GB_STORAGE,
+			constants.FEATURE_NO_ADS,
+		],
+		getSignupFeatures: () => [ constants.FEATURE_EMAIL_SUPPORT_SIGNUP ],
+		getBlogSignupFeatures: () => [
+			constants.FEATURE_EMAIL_SUPPORT_SIGNUP,
+			constants.FEATURE_ALL_FREE_FEATURES,
+		],
+		getPortfolioSignupFeatures: () => [ constants.FEATURE_EMAIL_SUPPORT_SIGNUP ],
+		// Features not displayed but used for checking plan abilities
+		getHiddenFeatures: () => [ constants.FEATURE_AUDIO_UPLOADS ],
+		getInferiorHiddenFeatures: () => [],
+		getAudience: () => i18n.translate( 'Best for P2ers' ),
+		getBlogAudience: () => i18n.translate( 'Best for P2ers' ),
+		getPortfolioAudience: () => i18n.translate( 'Best for P2ers' ),
+		getStoreAudience: () => i18n.translate( 'Best for P2ers' ),
+		...getMonthlyTimeframe(),
+		availableFor: ( plan ) => includes( [ constants.PLAN_FREE ], plan ), //TODO: only for P2 sites.
```

Comment:
I guess we will have to create a `PLAN_P2_FREE` at some point, or?

Reply:
Yeah, it's a good question.

I've been thinking about this and also discussing with @lezama that we could perhaps reuse the WP.com Free one and just adding the condition "if it's a P2 site + a Free plan..." to places where necessary like I did in this patch.

However, that might bite us in the future unexpectedly...

On the other hand, WP.com Free plan might be kind of special now that I'm thinking about it.

We'll definitely have to revisit this once we start thinking about opening P2+ to production but I think it's safe for now (enabled only for dev and staging).